src/main/java/com/lunark/lunark/properties/controller/PropertyController.java

```diff
@@ -65,10 +67,18 @@ public ResponseEntity<List<PropertyResponseDto>> getAll(
         List<PropertyResponseDto> propertyDtos = properties.stream()
                 .map(PropertyDtoMapper::fromPropertyToDto)
                 .toList();
+        return new ResponseEntity<>(propertyDtos, HttpStatus.OK);
+    }
 
+    @GetMapping(value="/unapproved", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<PropertyResponseDto>> getAllUnapproved(SpringDataWebProperties pageable) {
+        List<Property> unapprovedProperties = propertyService.findAll().stream().filter(property -> !property.isApproved()).toList();
```

Comment:
This can be a simple service call.
```suggestion
        List<Property> unapprovedProperties = propertyService.findUnapproved().
```
Add findByApproved(boolean) to IPropertyRepository and then call that from findUnapproved.

Reply:
You're correct.